doc/installation.rst

```diff
@@ -43,6 +43,14 @@ Standalone installers
 Our standalone installers for Windows and macOS are available from Spyder 4.2 onwards.
 We recommend using this installation method on those platforms, but we offer several other options for Linux, advanced users and specific needs, so keep reading if that's the case for you.
 
+
+Downloading and installing
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To download the supported Spyder installer for your platform, simply click the appropriate link below.
+(For Linux, see the :ref:`anaconda_install_ref` section).
+Then, double-click the downloaded file and proceed through the steps (if a security warning pops up, you may need to click :guilabel:`Yes`, :guilabel:`OK`, :guilabel:`Allow`, or (on Windows) :guilabel:`More info` followed by :guilabel:`Run anyway`).
```

Comment:
Okay, thanks for making me aware of the issue on Mac. Could you provide a link to wherever the exact steps needed for mac are shown/documented (or detail that here), so I can include them? I'm guessing you're talking about [this](https://support.apple.com/en-us/HT202491)?

I'd also be curious to hear about what "the license issue" is given that was one of my areas of focus in Spyder in the past and there was at least one clear license violation I identified (using some GPL-licensed code that effectively requires Spyder be GPL) that Carlos refused to fix.

Just to be clear, was the above issue what you meant by "the workflow you are describing is not entirely correct" or is there something else I am missing?

As you'll likely agree, this sentence is rather long and syntactically complicated. Especially if we need to add more Mac instructions, we can split it off into a separate sentences, like this:

```suggestion
Then, double-click the downloaded file and proceed through the steps.
If a security warning pops up, you may need to click :guilabel:`Yes`, :guilabel:`OK`, :guilabel:`Open` or :guilabel:`Allow`.
On Windows, you may need to click :guilabel:`More info` followed by :guilabel:`Run anyway`, and on macOS, you may need to click :guilabel:`Open Anyway` under :menuselection:`Security & Privacy --> General` in System Preferences.
```

Reply:
Yeah. The workflow you described is not entirely correct and yes Im talking about the information in the link. Seems that the link has the more accurate workflow. We are currently working with Ryan on adding a Mac license, it should be done soon.

Reply:
Given the steps at the link I was able to find were a macOS-specific addition to what I described generically and for Windows, I was confused as to whether there was something incorrect about the latter, or just referring to that additional steps were warranted for some macOS versions.

In any case, did what I added in the suggestion address that issue, or is something more needed? If so, it would be helpful if you could make it as a specific suggestion, to ensure we're both on the same page and avoid any further confusion. Thanks.

I'm still not clear what you mean about a license problem, or how that relates to the Gatekeeper issue. @mrclary , could you help clarify the situation here?

Reply:
I cannot speak about licensing (GPL, MIT, etc.). The link that you pointed to refers to code signing and is the Gatekeeper functionality of macOS to prevent users from inadvertently using malicious software. We are hoping to address this issue with spyder-ide/spyder#16490.

Incidentally, in addition to going to preferences to allow opening an unidentified developer application, after the first attempt to open the application a user can also right-click on the application and select "Open" and the pop-up window shown in the link that allows opening the application will appear.

Reply:
> I cannot speak about licensing (GPL, MIT, etc.). The link that you pointed to refers to code signing and is the Gatekeeper functionality of macOS to prevent users from inadvertently using malicious software.

Yep, that was my understanding as well, which is why I was confused when licensing issues kept being brought into it. Seems this was just a miscommunication. Also, thanks for linking the issue as I'd previously requested, so I can keep track of this.

>  a user can also right-click on the application and select "Open" and the pop-up window shown in the link that allows opening the application will appear.

Yeah, I initially considered saying that instead, but didn't want to confuse the instructions. However, I ended up revising things further just now to describe the install and run process on the different platforms more clearly and explicitly, and added that alongside the System Preferences method.

Reply:
@mrclary Does what I have now look good to you? If so, feel free to mark this as resolved, thanks.